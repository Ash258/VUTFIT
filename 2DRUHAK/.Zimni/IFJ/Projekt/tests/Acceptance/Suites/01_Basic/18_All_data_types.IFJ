' All data types

scope
	dim a as integer = 10
	dim b as string = !"ahojy\n\055"
	dim c as double = 30.42
	dim d as double = 30.3e2
	dim e as double = 30.0-2
	dim f as boolean = true
	dim g as string = !"retezec\032s\032lomitkem\032\092\032a\010novym\035radkem"

	print a; b; c; d; e; f; g;
end scope
