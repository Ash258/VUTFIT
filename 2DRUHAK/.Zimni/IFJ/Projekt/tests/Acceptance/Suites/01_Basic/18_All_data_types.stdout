 10ahojy
7 30.42 3030 28trueretezec s lomitkem \ a
novym#radkem